### DISCUSSION TITLE
[BUILD IOS] Operation not permitted (in target 'CapacitorCordova' from project 'Pods')

### DISCUSSION THREAD

**Discussion by danbn-100295** [ionic-team/capacitor#3362] [Q&A]
Hey everyone
I try build ionic5 with capacitor,but u have this error on xcode

`Operation not permitted (in target 'CapacitorCordova' from project 'Pods)`

Here is my version info:
`Xcode : Version 11.0 (11A420a)`

`Capacitor :  "@capacitor/core": "^2.4.0", "@capacitor/ios": "^2.4.0",`

**Comment by gilsonviana** [comment]
Same issue here.

**Comment by jcesarmobile** [comment]
see https://github.com/ionic-team/capacitor/issues/3489
